index.js

```diff
@@ -205,7 +211,11 @@ var TruffleContractSchema = {
     // update schema version
     normalized.schemaVersion = pkgVersion;
 
-    return normalized;
+    if (options.validate) {
```

Comment:
> Truffle producing barely invalid ABIs

Truffle's tests, correct? Truffle itself only "produces" abis that come directly from Solidity (using my interpretation of the word "produces").